pom.xml

```diff
@@ -35,6 +35,17 @@
     <maven.compiler.source>${java.version}</maven.compiler.source>
     <maven.compiler.target>${java.version}</maven.compiler.target>
     <maven.version>3.5.4</maven.version>
+    <build.testJarPhase>prepare-package</build.testJarPhase>
+    <CodeCacheSize>512m</CodeCacheSize>
+    <scala.version>2.12.10</scala.version>
+    <scala.binary.version>2.12</scala.binary.version>
+    <slf4j.version>1.7.16</slf4j.version>
+    <guava.version>14.0.1</guava.version>
+    <codahale.metrics.version>3.2.6</codahale.metrics.version>
```

Comment:
3.2.6 is a very old version and why do we use this version? The new version is 4.2.10
Ref: https://github.com/dropwizard/metrics/releases?page=1
https://metrics.dropwizard.io/4.2.0/getting-started.html